src/hello.cpp

```diff
@@ -0,0 +1,10 @@
+#include "hello.hpp"
+
+// Standalone function (because it's not a class)
+NAN_METHOD(hello) {
+
+    // info comes from the NAN_METHOD macro, which returns differently
+    // according to the version of node
+    info.GetReturnValue().Set(Nan::New<v8::String>("world").ToLocalChecked());
+
+}
```

Comment:
@springmeyer possible to add another Method in here? Or is the concept of "standalone function" that there is only _one_ function as part of this `cpp` file?